package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "@verida/wallet-utils",
-  "version": "1.4.0",
+  "version": "1.5.0",
   "description": "Wallet utilities for managing accounts, keys and signatures across multiple blockchains",
-  "main": "dist/utils.js",
+  "main": "dist/MultiChainWallets.js",
```

Comment:
Why change the entry point for the library?

Can we either have an `index` entry point that exposes both `MultiChainWallets` and `utils` or leave the entry point as `utils` and expose `MultiChainWallets` via `utils`?

We can't lose the ability to create or use a utils for a specific chain.

Reply:
Yep, you are right, I have fixed the entry point to be index.js and exposing both utils and multichainwallet